get-request.md

```diff
@@ -5,6 +5,14 @@ The GET gives the customer to access the burger meal section of the online menu
 | --- | --- |
 | GET | base_url://lunch/bugerMeal |
 
+
+
+The GET/mealType retrives the meal that is currently available, the burgerMeal.  
+|Attribute | Data Type | Description |
+| --- | ---| --- |
+| mealType | string | The meal that you order |
+| timestamp | date | the time that you order the request |
+
 ## GET Request Burgermeal
 
 Retrieves the meal that is available at the current time.  
```

Comment:
```suggestion
Retrieves the order/orders that are available at the current time.  An order is available if it has been placed, and has not been completed. If there is more than one order available, each order can be identified by the order number. 
```

Reply:
Also what happened to the tableNumber property?
You need to specify the table number in order to get the request.